src/Rendering/edgesRenderer.ts

```diff
@@ -149,6 +151,32 @@ export class EdgesRenderer implements IEdgesRenderer {
     /** Gets or sets a boolean indicating if the edgesRenderer is active */
     public isEnabled = true;
 
+    private static _LineShaders: Array<ShaderMaterial> = [];
+    private static _LineShaderScenes: Array<Scene> = [];

```

Comment:
It might be simpler to keep track of it on the scene  directly ???

Reply:
Done.